src/index.js

```diff
@@ -0,0 +1,16 @@
+import './style.css';
+import { renderTodos } from './functions.js';
+
+document.querySelector('.text-input').addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    // addTodo(e.target.value);
+    renderTodos();
+    e.target.value = '';
+  }
+});
+
+// document.querySelector('.clear-btn').addEventListener('click', () => {
+//   deleteAll();
+// });
+
```

Comment:
- [ ] Great job removing the parts of your code that weren't relevant for this milestone as the previous reviewer requested. However, you left them in your code as comments and this is not a good practice. So I will suggest that you remove all the commented pieces of code in your project. It makes your codebase unclean and is not a good practice :+1: